src/main/kotlin/io/sip3/salto/ce/sdp/SdpHandler.kt

```diff
@@ -159,12 +163,17 @@ class SdpHandler : AbstractVerticle() {
         return codec
     }
 
+    private fun definePtime(request: MediaDescriptionField, response: MediaDescriptionField): Int? {
```

Comment:
Let's just delete `defineCodec` and `definePtime` and use lazy `codec` and  `ptime` within `SdpSessionDescription` itself. It will be cleaner